lib/src/main/java/com/thenewboston/account/model/Account.kt

```diff
@@ -1,4 +1,11 @@
 package com.thenewboston.account.model
 
-// todo implement the proper model, check Account Manager repository
-data class Account(val placeholder: String)
\ No newline at end of file
+import org.apache.commons.lang3.StringUtils
+
+data class Account(val id: String, val balance: Balance, val balanceLock: BalanceLock) {
+    init {
+        require(StringUtils.isNotBlank(id)) {
+            "id must not be empty"
```

Comment:
Why not just use the Kotlin string utils extension function?

`id.isNotBlank()`
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/is-not-blank.html

No need for apache commons library.

Reply:
Good call. I had to remove validation logic from domain object. After discussing pr with @MrIceman we came to conclusion that we want DOs to be light.